lib/create-state.ts

```diff
@@ -3,10 +3,10 @@ import { Item } from "../types/item";
 import { getRandomItem, preloadImage } from "./items";
 
 export default async function createState(deck: Item[]): Promise<GameState> {
-  const next = getRandomItem(deck, []);
-  const nextButOne = getRandomItem(deck, []);
-  const imageCache = [preloadImage(next.image), preloadImage(nextButOne.image)];
   const played = [{ ...getRandomItem(deck, []), played: { correct: true } }];
+  const next = getRandomItem(deck, played);
+  const nextButOne = getRandomItem(deck, [...played, next]);
+  const imageCache = [preloadImage(next.image), preloadImage(nextButOne.image)];
```

Comment:
Unlike the first change, I feel like the order of operations does affect the outcome here. Would that be correct?

Reply:
Yes, 'getRandomItem' is now using 'played', so it's after its initialization.